packages/zoe/src/contracts/multipoolAutoswap/getCurrentPrice.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param {(brand: Brand) => boolean} isSecondary
+ * @param {(brand: Brand) => boolean} isCentral
+ * @param {(brand: Brand) => Pool} getPool
+ */
+
+import '../../../exported';
```

Comment:
Move above the `@param` declarations, since they describe `makeGetCurrentPrice`.